tests/Datagrid/PagerTest.php

```diff
@@ -770,4 +787,15 @@ protected function callMethod($obj, string $name, array $args = [])
 
         return $method->invokeArgs($obj, $args);
     }
+
+    /**
+     * NEXT_MAJOR: remove this method.
+     */
+    private function setProperty(object $obj, string $name, $value)
+    {
+        $class = new \ReflectionClass($obj);
```

Comment:
We could use a closure instead of the reflection API for this purpose.

Reply:
Is it worth it changing now? What do we gain from that?

Reply:
Not a blocker at all.
I was suggesting the change because we don't need any of the details provided by the reflection API here. We just need to get access to a protected property.
The gain we should get is performance. See https://ocramius.github.io/blog/accessing-private-php-class-members-without-reflection/.

Reply:
> The gain we should get is performance

yeah but since we execute this a couple of times its barely measurable anyway. So I would just leave it since we will remove it on master/4.x anyway.